invenio_mail/ext.py

```diff
@@ -16,7 +16,7 @@
 from . import config
 
 
-def print_email(message, app):
+def print_email(app, message):
```

Comment:
seeing how flask mail implements it, should it follow their signature?
```suggestion
def print_email(app, message=message):
```

Reply:
Not sure to understand the suggested change: Flask-Mail will call `print_email(app, message=message)`, but we cannot use `message=message` in a method definition.
The way this Pull Request is implemented follows the official [Signalling support](https://flask-mail.readthedocs.io/en/latest/#signalling-support) documentation, so it looks fine to me.